hub/api/info.py

```diff
@@ -14,14 +14,16 @@ def __init__(self):
         self.is_dirty = False
 
     def __enter__(self):
+        from hub.core.tensor import Tensor
+
         ds = self._dataset
         key = self._key
         if ds is not None:
             ds.storage.check_readonly()
             if not ds.version_state["commit_node"].is_head_node:
                 raise InfoError("Cannot modify info from a non-head commit.")
             if key:
-                ds[key].chunk_engine.commit_diff.modify_info()
+                Tensor(key, ds).chunk_engine.commit_diff.modify_info()
```

Comment:
why was this changed?

Reply:
tensor names have to be passed to __getitem__ of datasets, not keys.